### DISCUSSION TITLE
Linksys WRT3200ACM interfaces

### DISCUSSION THREAD

**Discussion by applamosca** [Ysurac/openmptcprouter#2153] [Q&A]
on the Linksys WRT3200ACM i can't set interfaces interfaces in bridge someone can help me
![tutorial](https://user-images.githubusercontent.com/12274601/140072988-4408b30a-f954-4940-9b21-d22facfbaad4.jpg)

**Comment by mdiaco** [comment]
select in type Bridge mode
